### PR TITLE
Added the PlayStation name properly

### PR DESCRIPTION
**Pull request by smitdylan2001**
Not yet changed in the localization files. If this needs to be done, let me know as well.

### CODE REVIEW

File: pcsx2/MMI.cpp [hunk 1]
```diff
@@ -1052,7 +1052,7 @@ static __fi void _PMADDW(int dd, int ss)
 	s64 temp = ((s64)cpuRegs.GPR.r[_Rs_].SL[ss] * (s64)cpuRegs.GPR.r[_Rt_].SL[ss]);
 	s64 temp2 = temp + ((s64)cpuRegs.HI.SL[ss] << 32);
 
-	//Playstation 2 division voodoo, for some reason only the lower half is affected
+	//PlayStation 2 division voodoo, for some reason only the lower half is affected
```
Comment (RedDevilus) [inline pcsx2/MMI.cpp:1055]: Might as well add a space between _// PlayStation_

Comment (smitdylan2001) [reply pcsx2/MMI.cpp:1055]: Wasn't focussing on punctuation tbh, just correctness of the console name

**Comment by seta-san** [comment]
If you make the playstation name properly do you have to add the tm mark?

**Comment by smitdylan2001** [comment]
> If you make the playstation name properly do you have to add the tm mark?

If you guys don't want it, that's fine, just close the PR.
On Sony's site it also says PlayStation. I just think it might be good to have it consistent. Right now some text does have the capital S and some don't.

**Comment by RedDevilus** [comment]
@seta-san That won't be necessary. PlayStation is in fact the correct name.

**Comment by Florin9doi** [comment]
can you fix "GitHib" too? 
```diff
diff --git a/pcsx2/gui/MainFrame.cpp b/pcsx2/gui/MainFrame.cpp
index 0c5285f55..c7b734047 100644
--- a/pcsx2/gui/MainFrame.cpp
+++ b/pcsx2/gui/MainFrame.cpp
@@ -506,7 +506,7 @@ void MainEmuFrame::CreateHelpMenu()
        m_menuHelp.Append(MenuId_Help_Website, _("&Website"));
        m_menuHelp.Append(MenuId_Help_Wiki, _("&Wiki"));
        m_menuHelp.Append(MenuId_Help_Forums, _("&Support Forums"));
-       m_menuHelp.Append(MenuId_Help_Github, _("&Github Repository"));
+       m_menuHelp.Append(MenuId_Help_Github, _("&GitHub Repository"));
        m_menuHelp.AppendSeparator();
        m_menuHelp.Append(MenuId_About, _("&About..."));
 }
```

**Review by RedDevilus — APPROVED** [review]
Looks good to me.

**Review by lightningterror — CHANGES_REQUESTED** [review]
For the translation files (po) you will need to use poedit and auto generate mo files.

**Comment by smitdylan2001** [comment]
The original English version has the wrong naming. I cannot edit these, so got the latest translation files from master

**Comment by refractionpcsx2** [comment]
If you can rebase and resolve the conflicts, are you done with this then?

**Comment by Florin9doi** [comment]
This PR is labeled Need Rebase but I can't see any conflict. Is there anything else needed?

**Comment by refractionpcsx2** [comment]
Well I can't merge it right now because of this

image

**Comment by lightningterror** [comment]
I think it can be squash merged instead if you want to avoid rebase merge conflicts.

**Comment by refractionpcsx2** [comment]
> I think it can be squash merged instead if you want to avoid rebase merge conflicts.

if you think that'll work, you can deal with it then ;)

**Comment by lightningterror** [comment]
I'd rather not, not until it's rebased properly just in case.

**Comment by smitdylan2001** [comment]
> If you can rebase and resolve the conflicts, are you done with this then?

Conflicts are fixed now

**Comment by RedDevilus** [comment]
Looks good except why did you re-add AppVeyor?

**Comment by smitdylan2001** [comment]
To add GitHub instead of github. 
Imo this small stuff should just be done when making a file, especially when working on something big.
If I can't change this file I can revert it back

**Comment by xddxd** [comment]
I believe RedDevilus meant that appveyor.yml is no longer in the master repo.

**Comment by smitdylan2001** [comment]
My bad, did not see this. Removed it again

**Comment by seta-san** [comment]
Merge this unholy pr before anything else changes.

**Comment by smitdylan2001** [comment]
If there is anything I can improve on, do let me know. I am still learning git.

**Review by RedDevilus — APPROVED** [review]
This should be ready, LT can handle the squashing and merge this.

File: 3rdparty/wxwidgets3.0/.travis.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # This is the control file for Travis continuous integration system.
 #
-# It is used automatically for the repositories on Github if it's found in the
+# It is used automatically for the repositories on GitHub if it's found in the
```
Comment (lightningterror) [inline 3rdparty/wxwidgets3.0/.travis.yml:3]: 3rdparty shouldn't be touched.

File: pcsx2/ps2/Iop/PsxBios.cpp [hunk 3]
```diff
@@ -58,7 +58,7 @@ void psxBiosReset()
     flush_stdout(true);
 }
 
-// Called for Playstation BIOS calls at 0xA0, 0xB0 and 0xC0 in kernel reserved memory (seemingly by actually calling those addresses)
+// Called for PlayStation BIOS calls at 0xA0, 0xB0 and 0xC0 in kernel reserved memory (seemingly by actually calling those addresses)
```
Comment (lightningterror) [inline pcsx2/ps2/Iop/PsxBios.cpp:61]: Is there really a need to update code comments?

File: pcsx2/MMI.cpp (hunk 1 above)
Comment (lightningterror) [inline pcsx2/MMI.cpp:1055]: Here as well.

File: linux_various/pre-commit [hunk 4]
```diff
@@ -7,7 +7,7 @@
 
 # modifications for clang-format by rene.milk@wwu.de
 # This file is part of a set of unofficial pre-commit hooks available
-# at github.
+# at GitHub.
```
Comment (lightningterror) [inline linux_various/pre-commit:10]: This too.

File: pcsx2/ps2/Iop/PsxBios.cpp (hunk 3 above)
Comment (tadanokojin) [reply pcsx2/ps2/Iop/PsxBios.cpp:61]: Why not? I'm not bothered.

**Comment by smitdylan2001** [comment]
I changed comments because I changed them fairly automatic, so why not keep them.
Also changed back the 3rd party one

**Review by tadanokojin — APPROVED** [review]

**Review by lightningterror — APPROVED** [review]

**Pull request merged**